### PR TITLE
Closes #34: Implement API endpoints for TodoList

### LINKED ISSUE #34: Implement API endpoints for the TodoList widget
**Is your feature request related to a problem? Please describe.**
As a front end developer, I want the API endpoints needed for the TodoList widget to be implemented, so I can interact with the back end.

**Describe the solution you'd like**
Implement the API endpoints needed for the TodoList widget.

**Describe alternatives you've considered**
None.

**Additional context**
These endpoints should be implemented by the back end team.

### PR DESCRIPTION
**Pull request by kyzh0**
Closes #34 
Create connection between todo api and db model

### CODE REVIEW

**Review by flexzy — CHANGES_REQUESTED** [review]
The APIs look good! Just need some tests for this endpoint before we can merge this. Have a look at the tests in the integration folder to see how we have started to test the API and database connections.

File: telos-backend/test/integration/todo_api.test.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  expect(todotasks.length).toBe(2);
+
+  // Check task2 is the same
+  expect(todotasks[0]._id).toStrictEqual(task2._id);
+  expect(todotasks[0].name).toBe(task2.name);
+  expect(new Date(todotasks[0].createdDate)).toStrictEqual(new Date(task2.createdDate));
+  expect(new Date(todotasks[0].dueDate)).toStrictEqual(new Date(task2.dueDate));
+  expect(todotasks[0].completed).toBe(task2.completed);
+
+  // Check event3 is the same
```
Comment (khal745) [inline telos-backend/test/integration/todo_api.test.js]: Nit: Spelling error, should be task3

**Review by khal745 — APPROVED** [review]
LGTM! One spelling error in a comment but not a big deal.

**Review by flexzy — APPROVED** [review]
LGTM! Ran the tests on my machine and they all passed. Logic looks legit. Nice work!

File: telos-backend/test/integration/todo_api.test.js (hunk 1 above)
Comment (kyzh0) [reply telos-backend/test/integration/todo_api.test.js]: Fixed, thanks!

**Pull request merged**